package-lock.json

```diff
@@ -0,0 +1,1459 @@
+{
```

Comment:
I suspecting that the lock file should be kept out of the VCS in this case.

What do you think?

Reply:
Agree